kele.el

```diff
@@ -1389,6 +1412,90 @@ Defaults to the currently active context as set in
   :options (lambda ()
              (alist-get 'group-versions (oref transient--prefix scope))))
 
+(cl-defun kele--get-context-arg ()
+  "Get the value to use for Kubernetes context."
+  (if-let* ((cmd transient-current-command)
+            (args (transient-args cmd))
+            (value (transient-arg-value "--context=" args)))
+      value
+    (kele-current-context-name)))
+
+(cl-defun kele--get-namespace-arg ()
+  "Get the value to use for Kubernetes namespace."
```

Comment:
Expand docstring to explain the order of precedence of where the value comes from.